mdpow/forcefields.py

```diff
@@ -180,163 +229,228 @@ def get_water_model(watermodel=DEFAULT_WATER_MODEL):
         raise ValueError(msg)
 
 
+@dataclass
+class Forcefield:
+    """Contains information about files corresponding to a forcefield."""
+
+    name: str
+    solvent_models: Dict[str, GromacsSolventModel]
+    forcefield_dir: Path
+    ions_itp: Path
+    default_water_itp: Path
+    default_water_model: str = "tip4p"
+
+    def __post_init__(self):
+        """Check that the provided paths exist and populate the :attr:`water_models`."""
+        for path_ in self.ff_paths:
+            if not path_.exists():
+                raise ValueError(f"Could not find required forcefield files: {path_}")
+
+        self.water_models: Dict[str, GromacsSolventModel] = _create_water_models(
+            self.forcefield_dir / "watermodels.dat"
+        )
+
+    @property
+    def ff_paths(self) -> List[Path]:
+        """Get the path to the forcefield directory and the ITP files for the ions and default water model."""
+        return [self.forcefield_dir, self.ions_itp, self.default_water_itp]
+
+    def __repr__(self) -> str:
+        """Get the name of the force field."""
+        return self.name
+
+
 #: Other solvents (not water, see :data:`GROMACS_WATER_MODELS` for those).
-new_octanol = """Zangi R (2018) Refinement of the OPLSAA force-field
-                for liquid alcohols.; ACS Omega 3(12):18089-18099.
-                doi: 10.1021/acsomega.8b03132"""
+NEW_OCTANOL_DESC = "Zangi R (2018) Refinement of the OPLSAA force-field for liquid alcohols.; ACS Omega 3(12):18089-18099.  doi: 10.1021/acsomega.8b03132"
+
+OPLS_AA_FF_DIR = TOP_DIR / "oplsaa.ff"
```

Comment:
Instead of the HERE approach, use
```suggestion
OPLS_AA_FF_DIR = Path(config.topfiles["oplsaa.ff"])
```
because in this way we control all access to bundled files from `mdpow.config`.